lib/operations/map_reduce.js

```diff
@@ -67,15 +68,15 @@ class MapReduceOperation extends OperationBase {
       reduce: reduce
     };
 
+    if (options['scope']) {
```

Comment:
I'm surprised eslint isn't catching this, but prefer `options.scope` to `options['scope']` for words that don't require quotation